lib/anyway/config.rb

```diff
@@ -127,11 +130,41 @@ def config_attributes
       end
 
       def required(*names)
-        unless (unknown_names = (names - config_attributes)).empty?
-          raise ArgumentError, "Unknown config param: #{unknown_names.join(",")}"
+        unknown_names = names - config_attributes
+        env_option = unknown_names.find { |name| env_attr?(name) }
+        raise ArgumentError, "Unknown config param: #{unknown_names.join(",")}" if unknown_names.any? && env_option.nil?
```

Comment:
I can't see spec for this change (we added `&& env_option.nil?`); am I missing something?

(I seems that there are not all edge cases covered right now in this guard)